src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,8 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        System.out.println("Hello World");
```

Comment:
The message printed to the console should be more informative, indicating what type of machine is working. Since this is a generic 'Hello World' message, consider updating it to reflect the operation of a specific machine such as a 'Truck', 'Bulldozer', or 'Excavator'.